src/VocabularyManager.Infrastructure/Services/OxfordDictionaryParser.cs

```diff
@@ -1,15 +1,16 @@
 ﻿using AngleSharp;
 using AngleSharp.Dom;
-using DictionaryParser.Exceptions;
-using DictionaryManager.Shared.Models.DTOs;
+using VocabularyManager.Core.Entities;
+using VocabularyManager.Infrastructure.Exceptions;
+using VocabularyManager.Core.Interfaces;
 
 namespace DictionaryParser.Services
 {
-    public class OxfordDictionaryParser : IWordParser<WordDTO>
+    public class OxfordDictionaryParser : IWordParser<Word>
```

Comment:
its in namespace `Services` but its `Parser`. Is it correct?

Reply:
You are right, missed one during restructuring